### DISCUSSION TITLE
Barrel Roll mod shows too many decimal digits in the rpm speed

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#17528] [Q&A] [answered]
I tried to search for similar issues about mods showing too many decimal digits, and the only thing I could find was [this discussion](https://github.com/ppy/osu/discussions/16750) (which, however, only referred to the Flashlight mod).

The issue here is that the Barrel Roll mod seems to do the same with its rpm speed.

Here's an example setting it to `2.76`:
![1](https://user-images.githubusercontent.com/43073074/160611463-9d858beb-0330-4dca-8e35-1297c0d9ff34.png)
![2](https://user-images.githubusercontent.com/43073074/160611490-2fceb607-8c68-46d8-be53-a16ec3df33eb.png)

At the moment I don't think I have found other mods with the same problem.

**Comment by frenzibyte** [comment, marked as answer]
#17529
